gsl_simulation/static/js/simulationProjetAcceptConfirmation.js

```diff
@@ -1,32 +1,42 @@
 let selectedElement = null;
 
 function handleStatusChange(select) {
-    if (select.value === "valid") {
-        showConfirmationModal(select);
+    if (["valid", "cancelled"].includes(select.value)) {
+        showConfirmationModal(select, select.value);
     } else {
         htmx.trigger(select.form, 'status-confirmed');  // Déclenche le PATCH HTMX
     }
 }
 
-function showConfirmationModal(select) {
+function showConfirmationModal(select, status) {
+    switch (status) {
+        case "valid":
+            modalId = 'accept-confirmation-modal';
+            break;
+        case "cancelled":
+            modalId = 'refuse-confirmation-modal';
+            break;
+    }
```

Comment:
Je voulais suggérer un objet de mapping pour alléger un peu la lecture, mais je ne sais même pas/plus si en JS je peux faire quelque chose du genre `modalIdFromStatus[status]`… Laissons comme ça pour l'instant ^^

Reply:
Il faudrait quand même gérer le cas où on a un status invalide ou vide, et quitter la fonction. 
```suggestion
function showConfirmationModal(select, status) {
    switch (status) {
        case "valid":
            modalId = 'accept-confirmation-modal';
            break;
        case "cancelled":
            modalId = 'refuse-confirmation-modal';
            break;
        default:
            console.error("ouin ouin");
            return;
    }
```

Reply:
T'as raison, j'étais pas très satisfait de ce switch case alors : 
```js
STATUS_TO_MODAL_ID = {
    "valid": "accept-confirmation-modal",
    "cancelled": "refuse-confirmation-modal"
}
```